OpenRA.Game/MiniYaml.cs

```diff
@@ -333,7 +333,7 @@ static List<MiniYamlNode> FromLines(IEnumerable<ReadOnlyMemory<char>> lines, str
 
 					// Note: We need to support empty comments here to ensure that empty comments
 					// (i.e. a lone # at the end of a line) can be correctly re-serialized
-					var commentString = comment == default ? null : comment.ToString();
+					var commentString = comment == ReadOnlySpan<char>.Empty ? null : comment.ToString();
```

Comment:
But [CA2265](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2265) says "don't compare to `default` because that's like comparing to `Span<T>.Empty`, use `.IsEmpty` instead"?

Reply:
In our case, this is the intended comparsion - we even trigger a unit test failure if we use `IsEmpty`. 

In our scenario this is more like a null comparison.
```miniyaml
Key: Value
Key: Value #
Key: Value # comment
```
In the first line we want `commentString` to be null - indicating no comment. In lines two and three we want it to be non-null, indicating there is a comment. On the second line, the comment happens to be empty, but it's still there.